hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1179,4 +1179,42 @@ public void testDatabaseLocationWithSlashInWarehouseDir() {
 
     assertThat(database.getLocationUri()).isEqualTo("s3://bucket/database.db");
   }
+
+  @Test
+  public void testListNamespacesWithEmptyNamespace() {
+    Namespace namespaceWithOneLevel = Namespace.of("parent");
+    assertThat(catalog.listNamespaces(namespaceWithOneLevel))
```

Comment:
at this point `namespaceWithOneLevel` is a non-exisiting namespace, so I think you should first create that namespace